charts/prometheus-adapter/templates/external-metrics-apiservice.yaml

```diff
@@ -1,7 +1,12 @@
 {{- if .Values.rules.external }}
-apiVersion: apiregistration.k8s.io/v1beta1
+apiVersion: apiregistration.k8s.io/v1
```

Comment:
does introduce breaking changes to older versions?

Reply:
You're right, I've reverted it back now :)

Reply:
You could add an if to check for the k8s version and use the api version available.

Example: https://github.com/prometheus-community/helm-charts/blob/main/charts/prometheus/templates/_helpers.tpl#L183-L186

Reply:
I thought about that, but there's already a PR open for it: https://github.com/prometheus-community/helm-charts/pull/463. I thought it best just to keep this PR as simple as possible for now.